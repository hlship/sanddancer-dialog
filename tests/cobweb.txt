(setup lighting)
teleport staging
drop lighter
burn web
take web
attack web
light web
x hole
take roll
(now) (knock holds #courage)
burn web
x hole
take roll 
x hole
