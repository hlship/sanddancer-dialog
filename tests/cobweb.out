

but it's too late, you're crashing, you crash...

Middle of Nowhere (in the pickup truck)
You've smashed up against a tall Saguaro, which is all you can see out the
windshield except a few feet of desert sand, then blackness. Hell, you must have
driven off the road and crashed. Ocean keeps telling you this night commute on
this lonely highway's gonna kill you; maybe you should start listening.

On the dash you can see a piece of jade.

The glove compartment is closed.

Through the windshield, you see a little lizard clinging frantically to the
Saguaro.

Something about the piece of jade makes you want to brood about your grandma's
stories.

> (setup lighting)
Query succeeded: (setup lighting)
> teleport staging

Staging Area
It's obvious this place has been abandoned for years. Cold night air breathes
through holes in the roof and everything is strewn with sand and patches of
mold. An imposing metal desk extends out from one wall. The floor has half-
collapsed near one corner, revealing a hole criss-crossed with cobwebs.

The boarded-up door is southwest, alongside the empty window frame leading
outside; other rooms lie north and east.

> drop lighter
Your lighter falls to the ground.

> burn web
(first attempting to take your lighter)
You take your lighter.

You think about reaching into the hole to light the cobwebs, then have a
nightmarish image of fat black spiders crawling over your hand. You shudder and
pull back.

> take web
You reach your hand towards the hole, then pull back with a start as you brush
cobwebs. Nightmare visions of fat black spiders shudder through your brain. No
way are you reaching in there.

> attack web
Yeah, sorry but you're not putting your hand in there.

> light web
You think about reaching into the hole to light the cobwebs, then have a
nightmarish image of fat black spiders crawling over your hand. You shudder and
pull back.

> x hole
Through a mass of cobwebs, you can see a roll of duct-tape down in the hole.

Hey, that roll of duct-tape might be useful if you decide on fixing the truck
and getting out of here.

> take roll
Yeah, sorry but you're not putting your hand in there.

> (now) (knock holds #courage)
> burn web
You flick your lighter and the cobwebs instantly ignite, almost instantly
shriveling to nothing.

> x hole
You can see a roll of duct-tape down in the hole.

> take roll 
You take the roll of duct-tape out of the hole.

> x hole
The hole is empty.

> 
